karma.conf.js

```diff
@@ -0,0 +1,48 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = function (config) {
+
+  // To specify browsers to test against use BROWSER var and comma separated list:
+  // BROWSER=Firefox,Chrome,Safari,IE npm run test:browser
+  var browsers = process.env.BROWSER ? process.env.BROWSER.split(',') : [ 'Firefox' ];
+
+  config.set({
+
+    browsers: browsers,
+    frameworks: [ 'mocha' ],
+    reporters: [ 'mocha' ],
+
+    files: [
+      'test/index.js'
+    ],
+
+    preprocessors: {
+      'test/index.js': [ 'webpack', 'sourcemap' ]
+    },
+
+    singleRun: true,
+
+    webpack: {
+      devtool: 'inline-source-map',
+      entry: path.join(__dirname, 'test', 'index.js'),
+      module: {
+        preLoaders: [
+          {
+            test: /\.js$/,
+            exclude: /node_modules/,
+            loader: 'babel',
+            query: {
+              presets: ['es2015'],
+              plugins: ['transform-object-assign']
```

Comment:
(I guess that also means that `--plugins transform-object-assign` should be removed from https://github.com/jlongster/redux-simple-router/blob/0941951a0a6948e526b33d49f49dfda033cbe6f0/package.json#L15, so maybe it should be its own PR?)